### PR TITLE
Refactors the rbac_processor to consolidate behavior and simplify testing

### PR DESCRIPTION
**Pull request by kthblmfld**
* removes modules named 'common'
* relocates 'common' features into specific modules
* establishes validators, state accessors, message (proto) accessors
* assigns unique names to functions
* accesses functions through their parent modules to avoid ambiguity

Signed-off-by: keithbloomfield <kthblmfld@gmail.com>

### CODE REVIEW

**Comment by kthblmfld** [comment]
Note: There is no new app functionality in this PR - just relocation, renaming of existing code for readability and testability. It might be easier to walk through the changes as a group rather than pour over the hundreds of renames, relocations in a code review.

File: processor/rbac_processor/main.py [hunk 1]
```diff
@@ -20,8 +20,9 @@
 from sawtooth_sdk.processor.core import TransactionProcessor
 from sawtooth_sdk.processor.log import init_console_logging
 
+from rbac_processor.event_handler import RBACTransactionHandler
 
-from rbac_processor.handler import RBACTransactionHandler
+HOST = os.getenv("HOST", "localhost")
```
Comment (chrisspanton) [inline processor/rbac_processor/main.py]: Duplicate lines at 25 & 27?

**Comment by adamgering** [comment]
He's cleaning up this PR for re-submission.

File: processor/rbac_processor/main.py (hunk 1 above)
Comment (kthblmfld) [reply processor/rbac_processor/main.py]: Yes. Removed the duplicate

**Pull request merged**